dynawo/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
```

Comment:
Please do not change the copyright of an existing file

Reply:
OK, I only changed the date; is there any reason ? What to do when there is no copyright ?

Reply:
All files should have a copyright, the year of creation needs to be given in the copyright when first added into the repository

Reply:
OK, new release is coming